api.planx.uk/saveAndReturn/utils.ts

```diff
@@ -1,12 +1,10 @@
 import { SiteAddress } from "@opensystemslab/planx-core/types";
 import { format, addDays } from "date-fns";
 import { gql } from "graphql-request";
-import {
-  adminGraphQLClient as adminClient,
-} from "../hasura";
+import { adminGraphQLClient as adminClient } from "../hasura";
 import { LowCalSession, Team } from "../types";
 import { Template, getClientForTemplate, sendEmail } from "../notify/utils";
-import { _admin } from "../client";
+import { _admin as $admin } from "../client";
```

Comment:
mind sharing reasoning for this syntax change?

Reply:
As @DafyddLlyr mentioned in the change that introduced `_admin` the underscore convention is generally used to indicate that a variable isn't referenced. I was working towards changing this shared planx-core client to `$admin` and `$public` instead if that works for everybody.